routes/index.js

```diff
@@ -337,11 +350,18 @@ module.exports = function (passport) {
       let adminEmail = req.user.email;
       let adminBadge = req.user.badge;
       dbAPI.logEvent('administration', adminBadge, `Admin ${adminName} (${adminEmail}) has deleted user with badge ID: ${deletedBadge}`);
-    });
-
-    req.flash('success', 'The user has been successfully deleted from the system');
-    req.flash('fade_out', '3000');
-    res.redirect('/userManagement');
+      dbAPI.logEvent('user traffic', deletedBadge, `User with badge ID: ${deletedBadge} has been deleted by Admin ${adminName}`);
+      req.flash('success', 'The user has been successfully deleted from the system');
+      req.flash('fade_out', '3000');
+      res.redirect('/userManagement');
+    }).catch(
+      err => {
```

Comment:
@danieleynis Do you mean like this? (above) I didn't run into that case in my testing; but I'd rather be safe than sorry. Good call.

Reply:
Yeah that looks good.